R/ml_model_utils.R

```diff
@@ -92,6 +100,11 @@ ml_generate_ml_model <- function(
   else
     ml_uid(predictor)
 
+# ml_features_names_metadata() just verifies the first stage (ml_stage(pipeline_model, 1)
+# and ml_lda() has more than one preprocessor
+if (grepl("lda", predictor$uid)) {
```

Comment:
It's better practice to check for `inherits(predictor, "ml_lda")` here since `uid` could be arbitrary (even though it's going to use the default in this case.)